modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ClusterCachesInfo.java

```diff
@@ -2514,7 +2572,7 @@ public void removeRestartingCaches() {
                 if (o1.cacheType().userCache() ^ o2.cacheType().userCache())
                     return o2.cacheType().userCache() ? -1 : 1;
 
-                return o1.cacheId().compareTo(o2.cacheId());
+                return o1.cacheId() - o2.cacheId();
```

Comment:
Undesirable change. Has almost the same logic as before, but produce errors on some corner cases. Not related to this ticket.

Reply:
It's better to use primitives in `MonitoringRow` for the sake of speed. 
Replaced with `Integer.compare(o1.cacheId(), o2.cacheId())`